src/components/NcPopover/NcPopover.vue

```diff
@@ -565,7 +565,11 @@ export default {
 			this.addEscapeStopPropagation()
 		},
 		afterHide() {
-			this.getPopoverContentElement().addEventListener('transitionend', () => {
+			/**
+			 * Component might be unmounted before 'apply-hide' is emitted.
+			 * Optional chaining is added to prevent errors in this case.
+			 */
+			this.getPopoverContentElement()?.addEventListener('transitionend', () => {
```

Comment:
> Component might be unmounted before 'apply-hide' is emitted 

This keeps it undetermined in which case it might happen and why it is expected and correct behavior.

And this is not possible. At the moment`<Dropdown>` emits `apply-hide` event, it is not mounted yet. Component instance is triggering the event. After unmounting, there is no instance to trigger the event.